src/theme/theme.ts

```diff
@@ -14,8 +14,7 @@ const theme = createTheme({
       contrastText: '#222222'
     },
     text: {
-      primary: '#222222',
-      secondary: '#FFFFFF'
+      primary: '#222222'
```

Comment:
secondary color here is used in input placeholder so in case you are using FFF color it will show white text  on white bg, so you will not see it